### DISCUSSION TITLE
custom the calendar icon size

### DISCUSSION THREAD

**Discussion by fadliasyhariStar** [Budibase/budibase#12749] [Help] [answered]
is there any way to make the calendar icon size bigger? Thanks in advance.

![image](https://github.com/Budibase/budibase/assets/134992316/2a00fee1-45e6-4935-ba68-90aaab0262c7)

**Comment by ConorWebb96** [comment, marked as answer]
Hey @fadliasyhariStar,

What browser are you using, your calendar is a lot smaller than the one which I'm looking at.

You should be able to use an embed component and use the below code within to force the calendar to go 80% full size of the box its within. **Make sure the embed component at the start of the component tree.**

```
<style>
svg.spectrum-Icon.spectrum-Icon--sizeM {
width: 100%;
}
</style>
```

Hopefully this helps.
